roles/edpm_ceph_hci_pre/tasks/firewall.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure firewall directory is present
+  ansible.builtin.file:
+    path: '/var/lib/edpm-config/firewall'
+    state: directory
+    owner: root
+    group: root
+    mode: 0750
```

Comment:
define the octal value as a string: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#parameter-mode

Reply:
done